auth-tokens/src/main/java/com/palantir/tokens/auth/UuidStringConverter.java

```diff
@@ -37,10 +37,9 @@ static String toString(UUID value) {
         final long msb = value.getMostSignificantBits();
         writeInt((int) (msb >> 32), ch, 0);
         ch[8] = '-';
-        int i = (int) msb;
-        writeShort(i >>> 16, ch, 9);
+        writeShort((int) msb >>> 16, ch, 9);
```

Comment:
please can you revert these changes? I find it difficult to read re: the precedence between (int) (msb >>> 16) and ((int) msb) >>> 16)

Reply:
casting to an int first or second doesn't matter in this case. I switched this to add explicit parenthesis. also switched the above `>>` to `>>>` since it's the only one of the four bit shifts that uses the signed shift (which again doesn't matter).